cmd/coder/sync.go

```diff
@@ -31,7 +32,8 @@ func (cmd *syncCmd) RegisterFlags(fl *pflag.FlagSet) {
 }
 
 // See https://lxadm.com/Rsync_exit_codes#List_of_standard_rsync_exit_codes.
-var NoRsync = errors.New("rsync: exit status 2")
+var errIncompatRsync = errors.New("rsync: exit status 2")
+var errStreamErrRsync = errors.New("rsync: exit status 12")
```

Comment:
One of the go linters complained these weren't named 'correctly'. Can change back to IncompatRsync, StreamErrRsync if we don't agree.